src/app/Library/Support/BackpackExceptionHandler.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace Backpack\CRUD\app\Library\Support;
+
+use Illuminate\Foundation\Exceptions\Handler;
+use Illuminate\Foundation\Exceptions\RegisterErrorViewPaths;
+use Illuminate\Support\Facades\Request;
+use Illuminate\Support\Str;
+
+class BackpackExceptionHandler extends Handler
+{
+    protected function registerErrorViewPaths()
```

Comment:
Is this the bit that makes it harmless? 👀🤷‍♂️ Because we're falling back to `RegisterErrorViewPaths` for non-backpack pages?